### PR TITLE
Add basic routes and scaffolding for better CSS management

### PR DESCRIPTION
**Pull request by simonv3**
# What's Included
* Make better routing
* Added some basic SASS scaffolding to better organize the styles. (Tell me if how I did it in the Grunt file makes sense).
* Created a button to match the view switcher in #3 to test the two things above out.
* Added click-through functionality on the world, countries, reports, etc.
* Some simple fixtures
* Refactoring to use ui-grid
* Setting up basic nettest pages. 

# What's next
* Get better data objects.

### CODE REVIEW

File: client/ngapp/index.html [hunk 1]
```diff
@@ -64,6 +77,7 @@
     <script src="scripts/app.js"></script>
     <script src="scripts/controllers/home.js"></script>
     <script src="scripts/controllers/report.js"></script>
+    <script src="scripts/controllers/explore.js"></script>
```
Comment (hellais) [inline client/ngapp/index.html]: You are missing here the import of the `country.js` controller.

File: client/ngapp/index.html [hunk 2]
```diff
@@ -64,6 +77,8 @@
...
+    <script src="scripts/controllers/country.js"></script>
```
Comment (simonv3) [inline client/ngapp/index.html]: @hellais just added it here.

File: common/models/report.js [hunk 3]
```diff
@@ -1,110 +1,43 @@
...
-    };
-    if (country_code && country_code.length) {
-      filter.where["probe_cc"] = {inq: country_code};
+  Report.countByCountry = function(alpha2, callback) {
+    var ds = Report.dataSource;
+    var sql = "SELECT probe_cc, count FROM country_counts_view";
+
+    // This is clearly not ideal, does Strongloop provide for sanitization?
+    // Surely their query language is a bit stronger than this?
+    if (alpha2 !== undefined && alpha2 !== null) {
```
Comment (simonv3) [inline common/models/report.js]: @hellais What are your thoughts on this? It doesn't seem very secure, but I wasn't sure whether Loopback does sanitization here.

**Comment by hellais** [comment]
I am going to proceed in merging this PR and suggest we continue all work on new feature branches.

**Pull request merged**